website/settings/defaults.py

```diff
@@ -283,31 +132,8 @@ def parent_dir(path):
     'node': [],
 }
 
-KEEN = {
-    'public': {
-        'project_id': None,
-        'master_key': 'changeme',
-        'write_key': '',
-        'read_key': '',
-    },
-    'private': {
-        'project_id': '',
-        'write_key': '',
-        'read_key': '',
-    },
-}
-
-SENTRY_DSN = None
-SENTRY_DSN_JS = None
```

Comment:
I don't see this in the api settings. @erinspace  Should this have been moved, too?

Reply:
Moved to api base defaults!